src/core/core.scale.js

```diff
@@ -734,32 +784,6 @@ var Scale = Element.extend({
 		return this.options.fullWidth;
 	},
 
-	// Get the correct value. NaN bad inputs, If the value type is object get the x or y based on whether we are horizontal or not
-	getRightValue: function(rawValue) {
```

Comment:
Add the removal of this method to the migration guide

Reply:
done